Dockerfile

```diff
@@ -41,6 +41,12 @@ RUN npm install
 
 COPY . $RAILS_ROOT/
 
+# Precompile assets with dummy values for secret key base and database connection (these will not be used elsewhere)
+ENV SECRET_KEY_BASE=55ca41d1c22ee9842aeef2aa5641c063f369e142723586958bd3b36921ca7026bdbf8b98a78376510bd351f790ed99c308e8cf5633b6e4ac678a0e8ef998f6dc
```

Comment:
why is this specified?

Can you make this come from the env but not specify a value for it here?  We can use a secret in github for builds there and local env var population elsewhere.